setup.py

```diff
@@ -8,12 +8,12 @@
 
 setup(
     name="elasticsearch-django",
-    version="5.2",
+    version="5.3-dev0",
     packages=find_packages(),
     install_requires=[
         'Django>=1.11',
-        'elasticsearch>=5',
-        'elasticsearch-dsl>=5',
+        'elasticsearch>=5,<6',
+        'elasticsearch-dsl>=5,<6',
```

Comment:
This is a breaking change - if you are currently using the 6.x packages this will roll them back to 5.x.